### PR TITLE
Add pages

### PR DESCRIPTION
**Pull request by kevinch**
Add base structure for all pages:

- Invoices export
- Invoices import
- Invoice
- Accounts export
- Accounts import
- Account
- Configuration

Live at https://kevin--qamarketplace.myvtexdev.com/admin/creditcontrol/invoices

### CODE REVIEW

File: react/account.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { injectIntl, intlShape } from 'react-intl'
+
+// eslint-disable-next-line
```
Comment (klzns) [inline react/account.js]: Pq?

File: react/accounts-import.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react'
+import { injectIntl, intlShape } from 'react-intl'
+import { Link } from 'render'
+
+// eslint-disable-next-line
```
Comment (klzns) [inline react/accounts-import.js]: Pq?

File: react/accounts.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react'
+import { injectIntl, intlShape } from 'react-intl'
+import { Link } from 'render'
+
+// eslint-disable-next-line
+class Accounts extends Component {
+  static contextTypes = {
```
Comment (klzns) [inline react/accounts.js]: Nao precisa. Recebe-se como prop

File: react/accounts.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        >
+          import
+        </Link>
+
+        <ul>
+          {is.map(i => (
+            <li key={i.id}>
+              <Link
+                className="underline-hover pointer"
+                page={'admin/creditcontrol-account'}
```
Comment (klzns) [inline react/accounts.js]: page="admin/creditcontrol-account"

File: react/config.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react'
+import { injectIntl, intlShape } from 'react-intl'
+
+// eslint-disable-next-line
+class Config extends Component {
+  static contextTypes = {
```
Comment (klzns) [inline react/config.js]: Recebe-se como prop

File: react/account.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+import PropTypes from 'prop-types'
+import { injectIntl, intlShape } from 'react-intl'
+
+class Account extends Component {
+  render() {
+    const { intl } = this.props
+
+    return (
+      <div>
+        <h1>{intl.formatMessage({ id: 'manage.title' })}</h1>
```
Comment (marcelotokarnia) [inline react/account.js]: You could probably use 

```
import {FormattedMessage} from 'react-intl'
<FormattedMessage id="manage.title" />
```

it has a cleaner syntax

File: react/accounts.js [hunk 7]
```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react'
+import { injectIntl, intlShape } from 'react-intl'
+import { Link } from 'render'
+
+class Accounts extends Component {
+  componentDidMount() {
```
Comment (marcelotokarnia) [inline react/accounts.js]: Now, every admin has to declare his own loading start and stop, 
you should do 

```
componentWillMount() {
  window.postMessage({action: {type: 'START_LOADING'}}, '*')
}
```

or something similar

File: react/account.js (hunk 6 above)
Comment (kevinch) [reply react/account.js]: Yes but it adds extra html markup.

File: react/accounts.js (hunk 7 above)
Comment (kevinch) [reply react/accounts.js]: Great, will update!

File: react/account.js (hunk 6 above)
Comment (marcelotokarnia) [reply react/account.js]: Yeah, that's right, if that's a concern you should leave it like this.

File: react/accounts.js (hunk 7 above)
Comment (kevinch) [reply react/accounts.js]: Issues created: https://github.com/vtex-apps/credit-control/issues/2

**Pull request merged**